### PR TITLE
Fix #5153: Growl severity for screen readers.

### LINKED ISSUE #5153: Growl: FacesSeverity not read out by screen readers
# Summary
There is a nice icon that represents FacesSeverity but there is no alternative text for it. I think primefaces should provide message keys per severity for i18n as well.

## 1) Environment
* PrimeFaces version: 7.0.6
* PrimeFaces showcase: https://www.primefaces.org/showcase/ui/message/growl.xhtml

## 2) Expected behavior
FacesSeverity is read in addition to message text.

## 3) Actual behavior
FacesSeverity alternative text is missing

## 4) Steps to reproduce
Produce a growl message and listen to the output of a screen reader.

## 5) Sample HTML from showcase
`<span class="ui-growl-image ui-growl-image-error"></span>`
`<div class="ui-growl-message">`
`<span class="ui-growl-title">Message: Validation Error: Value is required.</span><p>Message: Validation Error: Value is required.</p>`
`</div>`

## 6) Sample Fix
`<span class="ui-growl-image ui-growl-image-error"></span>`
`<span class="ui-helper-hidden-accessible">error</span>`
`<div class="ui-growl-message">`
`<span class="ui-growl-title">Message: Validation Error: Value is required.</span><p>Message: Validation Error: Value is required.</p>`
`</div>`

### PR DESCRIPTION
**Pull request by melloware**
I think this is a big improvement and allows IL8N of the severity to the screen reader.  its optional too so anyone not passing it in on the client side API it won't break anything.

### CODE REVIEW

**Comment by tandraschko** [comment]
+1

File: docs/7_1/core/accessibility.md [hunk 1]
```diff
@@ -15,15 +15,21 @@ provides English translations by default and you may use the following keys in y
 bundle to provide your own values.
 
 - primefaces.datatable.aria.FILTER_BY = Filter by {0}
-- primefaces.paginator.aria.HEADER = Pagination
+- primefaces.datatable.aria.HEADER_CHECKBOX_ALL = Select All
```
Comment (MarkusChur) [inline docs/7_1/core/accessibility.md:18]: there are changes not related to Growl.

File: src/main/java/org/primefaces/renderkit/UINotificationRenderer.java [hunk 2]
```diff
@@ -59,6 +61,32 @@ else if (ordinal == FacesMessage.SEVERITY_FATAL.getOrdinal()) {
         return severity;
     }
 
+    /**
+     * Gets the severity in IL8N human readable text for ARIA screen readers.
```
Comment (MarkusChur) [inline src/main/java/org/primefaces/renderkit/UINotificationRenderer.java]: I18N :)

**Comment by MarkusChur** [comment]
+1. Well done

**Pull request merged**